src/App.js

```diff
@@ -193,6 +193,37 @@ const App = compose(
             })
           }
         />
+        <Route
+          path={ROUTES.dashboard}
+          exact
+          render={(props) =>
+            protectRoute({
+              api,
+              isLoadingUser,
+              Component: UserDashboard,
+              loggedInUser,
+              ...props,
+            })
+          }
+        />
+
+        <Route
+          path={ROUTES.studies}
+          exact
+          render={(props) =>
+            protectRoute({
+              api,
+              isLoadingUser,
+              Component: () => {
+                import('pages/studies')
+                  .then(({ StudiesPage }) => StudiesPage)
+                  .catch(() => {});
```

Comment:
is the "catch" necessary?

Reply:
I'll add handling the catch clause in next update